src/styled.js

```diff
@@ -27,18 +33,25 @@ const styled = ({tagName, elementStyle, mountSheet}: StyledArgs) => {
   return class StyledElement extends Component {
     static tagName: string = tagName
     static style: ComponentStyleType = elementStyle
+    static contextTypes = {
+      [channel]: PropTypes.object
+    }
```

Comment:
yeah, I know, but you have `isThemingEnabled` because of you providing a themed-function instead an object.

so as I sad, this implementation is without themed-function, so I have just a plain object